test/rules/android_local_test/java_launcher_test.bzl

```diff
@@ -16,9 +16,9 @@
 launcher_test: Asserts that the executable is contained in the target's runfiles.
 """
 
-load("//rules:visibility.bzl", "PROJECT_VISIBILITY")
 load("@bazel_skylib//lib:sets.bzl", "sets")
 load("@bazel_skylib//lib:unittest.bzl", "analysistest", "asserts")
+load("//rules:visibility.bzl", "PROJECT_VISIBILITY")
```

Comment:
This may also be the result of a text transform from our export tool. Internally we have a monorepo, where the path for skylib is alphabetically before the path for visibility.bzl, and adjusting the order for the Github export would break the ordering inside the monorepo (not that it _really_ matters beyond pleasing the style checker tool).